trulens_eval/trulens_eval/keys.py

```diff
@@ -77,55 +172,182 @@ def get_huggingface_headers():
     return HUGGINGFACE_HEADERS
 
 
-def setup_keys(**kwargs):
-    global config_file
+def _value_is_set(v: str) -> bool:
+    return not(v is None or "fill" in v or v == "")
```

Comment:
I know its unlikely; but maybe we should check against a const 
USER_FILL_SUGGEST="to fill in..."

Two reasons
* There is a very low chance that the substring fill shows up in a key string
* Better code debuggability as this check is 2 methods down from the _check_or_set function of which this is the main branching for that method's behavior

Reply:
Did this.